config/crd/bases/infrastructure.cluster.x-k8s.io_azuremanagedmachinepools.yaml

```diff
@@ -297,6 +297,295 @@ spec:
                 - OS
                 - Temporary
                 type: string
+              linuxOsConfig:
```

Comment:
rename to `linuxOSConfig` to be consistent with azure api? ref: https://pkg.go.dev/github.com/Azure/azure-sdk-for-go@v58.1.0+incompatible/services/containerservice/mgmt/2021-05-01/containerservice#ManagedClusterAgentPoolProfile

Reply:
Updated!